stage1/usr_from_coreos/manifest.d/systemd.manifest

```diff
@@ -61,6 +61,9 @@ lib64/libmount.so.1
 lib64/libmount.so.1.1.0
 lib64/libnss_files-2.21.so
 lib64/libnss_files.so.2
+lib64/libpam.so
```

Comment:
Please.

Iago López Galeiras <notifications@github.com> schrieb am Mi., 27. Apr.
2016, 15:34:

> In stage1/usr_from_coreos/manifest.d/systemd.manifest
> <https://github.com/coreos/rkt/pull/2514#discussion_r61257134>:
>
> > @@ -61,6 +61,9 @@ lib64/libmount.so.1
> >  lib64/libmount.so.1.1.0
> >  lib64/libnss_files-2.21.so
> >  lib64/libnss_files.so.2
> > +lib64/libpam.so
>
> We only updated systemd in stage1 3 times, that's why we didn't spend much
> time on this.
>
> Right now my process is unpacking the CoreOS image (gunzip, cpio,
> unsquashfs), updating manually the versions of the libraries, trying to run
> rkt with that and adding more libraries as I get errors.
>
> Should I write that down somewhere? Maybe Documentation/hacking.md or a
> new Documentation/devel/update-coreos-stage1.md?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2514/files/8fed61d3c7b78d72433f8eb995995ffec51d08fe#r61257134>
>